mmv1/third_party/terraform/fwvalidators/framework_validators.go

```diff
@@ -52,21 +51,21 @@ func CredentialsValidator() validator.String {
 }
 
 // Non Negative Duration Validator
-type nonnegativedurationValidator struct {
+type nonnegativeBoundedDuration struct {
 }
```

Comment:
Please revert the changes to this validator - it doesn't make assertions about the range/bounds a duration should be in so this name is misleading. My feedback previously was about your validator and showing how this validator's name indicated how it validated the duration.

Reply:
reverted.